### DISCUSSION TITLE
How To Update UI

### DISCUSSION THREAD

**Discussion by MahmoudNasser01** [horilla-opensource/horilla#392] [General]
When I update something in css it is not reflected in the development

**Comment by JVKeller** [comment]
you would have to delete the cache and rebuild it.

**Reply by MahmoudNasser01** [reply]
even if I change in the Scss files ?

**Reply by JVKeller** [reply]
I don't know exactly, haven't tried that yet. When you update and save the scss, if the changes are not showing on a ctrl+F5 reload, then yeah, the cache would need to be rebuild.

**Reply by JVKeller** [reply]
btw...  from my experience, you're better off to set up overrides that editing the original scss. That way if you update a template, your changes will not get overwritten!

So unless you are planning to merge your changes, figure out how to do overrides!
